packages/app-spec-squad-chess/src/Board.js

```diff
@@ -144,7 +144,7 @@ const BoardSquare = {
     let squareContent
     let onclick
     // if the square holds a piece, set the properties
-    if (vnode.attrs.content) {
+    if (vnode.attrs.content && vnode.attrs.content.pieceId) {
```

Comment:
There were squares with content (promotable) that would later fail when looking for graphics. This now also checks to make sure that there is a piece to look for graphics on

Reply:
I think `promotoble` was meant to be outside of `content` -- so `content` would only be truthy if there's a piece, otherwise `null`. Is that not the case?

Reply:
Nope you are right, that's the case. resolving these conversations